atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -546,7 +547,8 @@ private void lowerConsistencyWhenSafe() {
         Map<String, String> strategyOptions;
 
         try {
-            dcs = clientPool.runWithRetry(client -> CassandraVerifier.sanityCheckDatacenters(client, config));
+            dcs = clientPool.runWithRetry(client -> CassandraVerifier.sanityCheckDatacenters(
+                    client, config.servers(), config.replicationFactor(), config.ignoreNodeTopologyChecks()));
```

Comment:
1) Yes, it uses less than half, so I decided that it didn't need the full keyspace verifier config (and was a motivating factor in the name - the keyspace verifier config is used in the methods that actually verify something about the keyspace). I'd briefly considered using a DatacentreVerifierConfig, but decided it didn't really add that much value.

2) Yeah, that's reasonable 👍

Reply:
2. I mean, this is the `CassandraVerifier`, so doesn't using the verifier config make sense? You could always just call it `CassandraVerifierConfig` or something and just pass it in; it seems like otherwise we don't really gain much here.